pkg/ibmvpc/parser.go

```diff
@@ -465,20 +502,41 @@ func getFipConfig(
 func getVPCconfig(rc *ResourcesContainer, res *vpcmodel.CloudConfig) {
 	for i := range rc.vpcsList {
 		vpc := rc.vpcsList[i]
-		vpcNodeSet := &VPC{VPCResource: vpcmodel.VPCResource{
-			ResourceName: *vpc.Name, ResourceUID: *vpc.CRN, ResourceType: "VPC"}, nodes: []vpcmodel.Node{}}
+		vpcNodeSet := &VPC{
+			VPCResource: vpcmodel.VPCResource{ResourceName: *vpc.Name, ResourceUID: *vpc.CRN, ResourceType: "VPC"},
+			nodes:       []vpcmodel.Node{},
+			zones:       map[string]*Zone{},
+		}
 		res.NodeSets = append(res.NodeSets, vpcNodeSet)
 		res.NameToResource[vpcNodeSet.Name()] = vpcNodeSet
 	}
 }
 
+func singleVPCErr() error {
+	return errors.New("expecting single vpc")
```

Comment:
This error message is a bit misleading. Should be "NACL/SG VPC (<vpc1>) is different from its layer's VPC (<vpc2>)."

Reply:
done